src/register.c

```diff
@@ -11,6 +11,11 @@
 static inline int do_register(struct io_uring *ring, unsigned int opcode,
 			      const void *arg, unsigned int nr_args)
 {
+	if (ring->features & IORING_FEAT_REG_REG_RING
+	    && ring->int_flags & INT_FLAG_REG_RING) {
```

Comment:
int_flags is not part of the API (I think?), and there is only 1 flag currently, so you could set a flag (say INT_FLAG_REG_REG_RING) for 
```
ring->features & IORING_FEAT_REG_REG_RING
	    && ring->int_flags & INT_FLAG_REG_RING
```

and then simplify the check here.

I think there is no real downside to this

Reply:
Done.